src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -96,6 +99,66 @@ public KafkaTopicRepository(final ZooKeeperHolder zkFactory,
         this.circuitBreakers = new ConcurrentHashMap<>();
     }
 
+    private CompletableFuture<Exception> publishItem(
```

Comment:
why you moved this method up? Now it's very difficult to see what was actually changed.

Reply:
it was wrongly formatted, and every time I went to this class it changes the sequence of lines, so I decided to commit it.